db/background_mgr_resync.go

```diff
@@ -61,9 +61,13 @@ func (r *ResyncManager) Run(ctx context.Context, options map[string]interface{},
 		persistClusterStatus()
 	}
 
-	_, err := database.UpdateAllDocChannels(ctx, regenerateSequences, callback, terminator)
-	if err != nil {
-		return err
+	for _, collection := range database.CollectionByID {
+		_, err := (&DatabaseCollectionWithUser{
```

Comment:
Should we be populating the DatabaseCollectionWithUser with database's user?
If not, should UpdateAllDocChannels be moved from DatabaseCollectionWithUser to DatabaseCollection?

I suspect it was defined on Database to simply calls to other Database methods, even if it always runs as admin.  If that's not worth refactoring right now, it feels like we should still respect any user that is potentially present on Database.